Reed/DefinitionView/viewmodels/DefinitionViewModel.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  DefinitionModalViewModel.swift
+//  Reed
+//
+//  Created by Hugo Zhan on 11/3/20.
+//  Copyright © 2020 Roger Luo. All rights reserved.
+//
+
+import SwiftUI
+
+class DefinitionViewModel: ObservableObject {
+    @Published var title = ""
+    @Published var currReading = ""
```

Comment:
General nitpick (marked with "nit:") feedback for this PR is that we don't have to be so conservative with keeping variable names short. Let's prioritize readability.